### PR TITLE
FE playwright setup: clone and remove kinetic specific code

### PR DESCRIPTION
**Pull request by Malar-Natarajan**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/1946

### CODE REVIEW

**Comment by Malar-Natarajan** [comment]
@gregfitch can you please review this draft PR. I have copied content from oslabs folder and removed most of the labs code and retained some mostly related to restmail. The rex.spec.ts is just sample, please ignore that.

Also restmail.ts & user.ts shows error that `axios` and `faker` are not installed. Do we need to install them separately?

File: playwright/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+*Testing requires access to [git](https://git-scm.com/downloads), [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm), and (for simple testing) [Docker](https://docs.docker.com/engine/install/).*
+
+### Clone the test repo
+
+`git clone https://github.com/gregfitch/oslabs-test`
+
+`cd oslabs-test`
+
```
Comment (gregfitch) [inline playwright/README.md:16]: update the readme markdown for REx

File: playwright/tests/rex-test/rex.spec.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
+
+test('open rex page', async ({ page }) => {
+    await page.goto('https://staging.openstax.org/books/physics/pages/preface');
+    expect(page.url()).toBe('https://staging.openstax.org/books/physics/pages/preface');
```
Comment (gregfitch) [inline playwright/tests/rex-test/rex.spec.ts:5]: Should use the base URLs instead (down the line, at least...)

File: playwright/README.md (hunk 1 above)
Comment (Malar-Natarajan) [reply playwright/README.md]: @gregfitch updated readme

File: playwright/tests/rex-test/rex.spec.ts (hunk 2 above)
Comment (Malar-Natarajan) [reply playwright/tests/rex-test/rex.spec.ts:5]: will do in the next PR while starting a real test

**Review by gregfitch — APPROVED** [review]

**Review by Dantemss — APPROVED** [review]
2 small comments but nothing major

**Pull request merged**